app/_config/docs-repositories.yml

```
@@ -162,3 +162,7 @@ RefreshMarkdownTask:
       - silverstripe/silverstripe-documentconverter
       - silverstripe-documentconverter
       - master
+    -
+      - silverstripe/ckan-registry
+      - ckan-registry
```

```suggestion
      - silverstripe-ckan-registry
```

For consistency with the other modules

---

A lot of the modules added here are named like this e.g 'elemental', 'taxonomy' and 'dsm'. This is just the folder name that the symlink pulls in, see https://github.com/silverstripe/silverstripe-userhelp-content/pull/133/files. I'll make a PR to change the inconsistent reqs (maintenance, sharedraftcontent, documentconverter).